src/elements/content-sidebar/versions/VersionsSidebar.js

```diff
@@ -49,8 +62,18 @@ const VersionsSidebar = ({ error, isLoading, parentName, versions, ...rest }: Pr
         >
             <LoadingIndicatorWrapper className="bcs-Versions-content" crawlerPosition="top" isLoading={isLoading}>
                 {showError && (
-                    <InlineError title={<FormattedMessage {...messages.versionServerError} />}>
+                    <InlineError title={<FormattedMessage {...errorTitle} />}>
                         <FormattedMessage {...error} />
+                        {showVersionHistoryErrorUpsell && (
```

Comment:
Can we make `onUpgradeCTAClick` nullable and use its presence/absence to control the visibility of the button? We would be able to eliminate the `showVersionHistoryErrorUpsell` prop entirely, if so.

Reply:
Ah, yes that makes sense! Making that change now!